src/components/forms/fields/FieldLayout/FieldLayout.module.scss

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@use '../../../../styling/defs.scss' as bk;
+
+@layer baklava.components {
+  .bk-field-layout {
+    @include bk.component-base(bk-field-layout);
+
+    &.bk-field-layout--small {
+      inline-size: calc(bk.$spacing-17 + bk.$spacing-6); // 244px
```

Comment:
Let's just replace this with `rem-from-px(244)`. In the future we probably need a separate set of variables for this. `spacing-` variables could be limited to just gaps/padding/margins etc.